app/controllers/conferences_controller.rb

```diff
@@ -2,17 +2,24 @@ class ConferencesController < ApplicationController
   protect_from_forgery with: :null_session
   before_action :respond_to_options
   load_and_authorize_resource find_by: :short_title
-  load_resource :program, through: :conference, singleton: true, except: :index
 
   def index
     @current = Conference.where('end_date >= ?', Date.current).reorder(start_date: :asc)
     @antiquated = @conferences - @current
   end
 
-  def show; end
+  def show
+    # have to change "localhost" to ENV['OSEM_HOSTNAME'] in production
+    check_custom_domain if request.host != 'localhost'
```

Comment:
At the top we already do `load_and_authorize_resource find_by: :short_title`, but in case I'm accessing a conference directly through its domain (ie. `www.myconf.com`), then `params[:short_title]` will be `nil`, right? What will `load_and_authorize_resource` do then? Maybe we should call `load_and_authorize_resource` only if we have `params[:short_title]`?